gen_bot_db.sh

```diff
@@ -40,8 +60,12 @@ printProjectJson() {
   echo '"id": '"${id}"','
   echo '"projectId": "'"${projectId}"'",'
   echo '"username": "genie.'"${projectShortName}"'"'
-  
-  if [[ -f "${projectPath}/email.gpg" ]]; then
+
+  ldap_email=$(query_email_by_user_id "genie.${projectShortName}")
+
+  if [[ "${ldap_email}" != "" ]]; then
+    echo ',"email": "'${ldap_email}'"'
+  elif [[ -f "${projectPath}/email.gpg" ]]; then
```

Comment:
I'd just remove this elif now that we have the api query. WDYT?

Reply:
Agreed.

Reply:
Fixed.

Reply:
Ok, I was not thinking removing every elif, but it can do. However, I think we should have a warning logged in the else case. WDYT?